src/pages/workspaces/workspace/JobHistory.js

```diff
@@ -159,44 +160,34 @@ const JobHistory = _.flow(
           },
           columns: [
             {
-              size: { basis: 600, grow: 0 },
+              size: { basis: 500, grow: 0 },
               headerRenderer: () => h(HeaderCell, ['Job']),
               cellRenderer: ({ rowIndex }) => {
                 const {
-                  methodConfigurationNamespace, methodConfigurationName, submitter, submissionId, workflowStatuses,
-                  status, submissionEntity
+                  methodConfigurationNamespace, submitter, submissionId, workflowStatuses
                 } = submissions[rowIndex]
                 return h(Fragment, [
                   div([
                     div([
                       methodConfigurationNamespace !== namespace && span({ style: styles.deemphasized }, [
                         `${methodConfigurationNamespace}/`
                       ]),
-                      methodConfigurationName
+                      link({
+                        target: '_blank',
+                        //replace link with actual job manager link
+                        href: linkToJobManager ? `https://jobs.terra.bio/${namespace}/${name}/${submissionId}`: `${getConfig().firecloudUrlRoot}/#workspaces/${namespace}/${name}/monitor/${submissionId}`
+                      }, ['methodConfigurationName'])
                     ]),
                     div([
                       span({ style: styles.deemphasized }, 'Submitted by '),
                       submitter
                     ])
                   ]),
-                  h(PopupTrigger, {
+                  collapsedStatuses(workflowStatuses).running && h(PopupTrigger, {
                     side: 'bottom',
                     closeOnClick: true,
                     content: h(Fragment, [
                       h(MenuButton, {
-                        as: 'a',
-                        target: '_blank',
-                        href: `${getConfig().firecloudUrlRoot}/#workspaces/${namespace}/${name}/monitor/${submissionId}`
-                      }, [menuIcon('circle-arrow right'), 'View job details']),
-                      isTerminal(status) && workflowStatuses['Failed'] &&
-                      submissionEntity && submissionEntity.entityType.endsWith('_set') && h(MenuButton, {
-                        onClick: () => rerunFailures({
-                          namespace, name, submissionId,
-                          configNamespace: methodConfigurationNamespace, configName: methodConfigurationName,
-                          onDone: () => this.refresh()
-                        })
-                      }, [menuIcon('sync'), 'Re-run failures']),
-                      collapsedStatuses(workflowStatuses).running && h(MenuButton, {
                         onClick: () => this.setState({ aborting: submissionId })
                       }, [
                         menuIcon('warning-standard', { style: { color: colors.orange[0] } }),
```

Comment:
The button/link to "Abort all workflows" is still in the submenu. Are we leaving that there? I think we should bring it out of the caret menu.
![screen shot 2019-02-04 at 4 19 33 pm](https://user-images.githubusercontent.com/2395211/52237969-e3976280-2898-11e9-9f2e-822f652e32e8.png)

Reply:
Yeah, I'm not sure what to do here. It's only there when the job is running, and I agree we should bring it out and eliminate the drop down entirely, just not really sure where to put it.

Maybe next to status similar to where we put the rerun failures?

Reply:
added an edit to the PR which moves the abort workflows next to status